.github/dependabot.yml

```diff
@@ -0,0 +1,54 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+  - package-ecosystem: "gomod"
+    directory: "/cert/"
+    schedule:
+      interval: "weekly"
+  - package-ecosystem: "gomod"
+    directory: "/clock/"
+    schedule:
+      interval: "weekly"
+  - package-ecosystem: "gomod"
+    directory: "/healthcheck/"
+    schedule:
+      interval: "weekly"
+  - package-ecosystem: "gomod"
+    directory: "/kvdb/"
+    schedule:
+      interval: "weekly"
+  - package-ecosystem: "gomod"
+    directory: "/queue/"
+    schedule:
+      interval: "weekly"
+  - package-ecosystem: "gomod"
+    directory: "/ticker/"
+    schedule:
+      interval: "weekly"
+  - package-ecosystem: "docker"
```

Comment:
Is there a way to tell dependabot to only alert about new docker base images? Because we want to keep the golang version we use in sync across several files so we update them with manual PRs. I think with dependabot it's going to be too noisy.

Reply:
I see two options:
* Removing the docker section in dependabot and dealing manually with image updates (happy to update this PR)
* Just interpreting the docker container update PRs as alerts and relying on dependabot to automatically close them as soon as they become obsolete by a different commit from somebody, that updates everything across the repo in various places.

I don't expect this becoming too noisy as soon as the initial batch of PRs got merged. If it still does, then things can be limited or disabled or dependabot can be silenced via commands in the according PRs to ignore certain versions.